tribe-ext-calendar-widget-areas/index.php

```diff
@@ -61,12 +56,14 @@ public function construct() {
 	 * to use. You will also need to add an additional method toward the end of this
 	 * file for each area added so we know what code to execute.
 	 * 
-	 * @access: public
-	 * @return: array
+	 * @return array
```

Comment:
Great thought on documenting the format for this array. Typically the documentation would go in the docblock. If you have a minute, perhaps you could move it to the return function here, and put it in the standard docblock format:
```
* @return array {
*     Every widget area supported by this extension
* 
*     @param array $action_hook {
*         The details for each widget area
* 
*         @param string $method The method of this class that outputs the widget
*         @param string $name   Name of the widget in the admin screens
*         @param string $desc   Widget description
*     }
* }
```